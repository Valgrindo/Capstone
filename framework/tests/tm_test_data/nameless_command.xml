<!-- Test that commands require a name -->
<commands>
    <command>
        <component word="TEST"/>
    </command>
</commands>