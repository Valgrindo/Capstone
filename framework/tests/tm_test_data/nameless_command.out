Command missing a name attribute.